backend/src/middleware/file_upload_security.py

```
@@ -0,0 +1,694 @@
+                else:
+                    # Threat detected
+                    result['errors'].append(
+                        f"Security threat detected: {scan_result}"
+                    )
+                    result['security_flags'].append('virus_detected')
+
+            finally:
+                # Clean up temporary file
+                try:
+                    os.unlink(temp_file_path)
+                except:
```

Ruff (0.14.0)

Do not use bare `except`

(E722)